src/adapters/configuration/adapter_config.py

```diff
@@ -499,6 +508,8 @@ class LoRAConfig(AdapterConfig):
     composition_mode: str = "add"
     init_weights: str = "lora"
     use_gating: bool = False
+    d: Union[bool, float] = None
+    b: Union[bool, float] = None
```

Comment:
could we name these "vera_b" and "vera_d", to make more obvious what these are related to?

Reply:
also why can these be bools? ie what happens when I set `d=True, b=True`?

Reply:
Good catch, i think this is a typo based on a previous idea I had which i scraped later.. thanks